crates/local-cluster-runner/src/node/mod.rs

```diff
@@ -143,32 +143,68 @@ impl Node {
             .build()
     }
 
-    // Creates a group of Nodes with a single metadata node "metadata-node", and a given number
-    // of other nodes ["node-1", ..] each with the provided roles. Node name, roles,
-    // bind/advertise addresses, and the metadata address from the base_config will all be overwritten.
-    pub fn new_test_nodes_with_metadata(
+    // Creates a group of Nodes with a single metadata node "metadata-node" running the
+    // metadata-store and the admin role, and a given number of other nodes ["node-1", ..] each with
+    // the provided roles. Node name, roles, bind/advertise addresses, and the metadata address from
+    // the base_config will all be overwritten.
+    pub fn new_test_nodes_with_admin(
         base_config: Configuration,
         binary_source: BinarySource,
         roles: EnumSet<Role>,
         size: u32,
     ) -> Vec<Self> {
+        Self::new_test_node_with_extra_node(
+            base_config,
+            binary_source,
+            enum_set!(Role::MetadataStore | Role::Admin),
+            roles,
+            size,
+        )
+    }
+
+    // Creates a group of Nodes with a single metadata node "metadata-node" running the
+    // metadata-store, and a given number of other nodes ["node-1", ..] each with
+    // the provided roles. Node name, roles, bind/advertise addresses, and the metadata address from
+    // the base_config will all be overwritten.
+    pub fn new_test_nodes_with_metadata_store(
+        base_config: Configuration,
+        binary_source: BinarySource,
+        roles: EnumSet<Role>,
+        size: u32,
+    ) -> Vec<Self> {
+        Self::new_test_node_with_extra_node(
+            base_config,
+            binary_source,
+            enum_set!(Role::MetadataStore),
+            roles,
+            size,
+        )
+    }
+
+    fn new_test_node_with_extra_node(
```

Comment:
Minor, I think this was meant to be:

```suggestion
    fn new_test_nodes_with_extra_node(
```